### PR TITLE
Add USWDS to app

### PR DESCRIPTION
**Pull request by Sgtpluck**

### CODE REVIEW

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -11,3 +11,6 @@ import "channels"
 Rails.start()
 Turbolinks.start()
 ActiveStorage.start()
+
+import 'uswds/dist/css/uswds.min.css'
+import 'uswds/dist/js/uswds.min.js'
```
Comment (jduss4) [inline app/javascript/packs/application.js:16]: I don't think we've talked yet about style guide preferences, etc, but I typically have come from a school of thought where all files end with a return line.  What are your thoughts on that?

Comment (Sgtpluck) [reply app/javascript/packs/application.js:16]: indifferent, so happy to implement it! let me see if there's a way to get out linter to yell at me about it too.

Comment (Sgtpluck) [reply app/javascript/packs/application.js:16]: (the answer for the linter is no, not really)

**Comment by Sgtpluck** [comment]
@jduss4 added the newline

**Review by jduss4 — APPROVED** [review]

**Pull request merged**